docs/book/helpers/doctype.md

```diff
@@ -45,15 +45,30 @@ And then print it out on top of your layout script:
 <?php echo $this->doctype() ?>
 ```
 
-Within an application based off the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton),
-you can specify the doctype via configuration:
+## Usage in a Mezzio Application
+
+`\Laminas\View\Helper\Service\DoctypeFactory` checks the application configuration, making it possible to
+define the doctype through your configuration (e.g. a `ConfigProvider`):
+
+```php
+return [
+    'view_helper_config' => [
+        'doctype' => 'HTML5',
+    ],
+];
+```
+
+## Laminas MVC View Manager
```

Comment:
```suggestion
## Usage in a laminas-mvc Application
```
We already use this wording in other places.

Reply:
argh damn. sorry, I already had it and then pasted the heading from the last paragraph over it 🤦‍♂️